regparser/grammar/terms.py

```diff
@@ -26,10 +26,15 @@
     LineStart()
     + Optional(Suppress(unified.any_depth_p))
     + e_tag.setResultsName("head")
+    + Suppress(ZeroOrMore(unified.any_depth_p))
```

Comment:
So, we have some terms that are defined like this:

a. Term
    1. Means blah